src/synapse-plugins/converter-plugin/converter-data.vala

```diff
@@ -0,0 +1,214 @@
+        {UnitType.DIMENSION, UnitSystem.IMPERIAL_UK, "gill", "", NC_(VOLUME, "gill"), "1/4", "ukpint"},
+        {UnitType.DIMENSION, UnitSystem.IMPERIAL_UK, "ukcup", "cup", NC_(VOLUME, "British cup"), "1/2", "ukpint"},
+        {UnitType.DIMENSION, UnitSystem.IMPERIAL_UK, "ukteacup", "teacup", NC_(VOLUME, "British teacup"), "1/3", "ukpint"},
+        {UnitType.DIMENSION, UnitSystem.IMPERIAL_UK, "uktbsp", "tbsp", NC_(VOLUME, "British tablespoon"), "15/1000", "liter"},
+        {UnitType.DIMENSION, UnitSystem.IMPERIAL_UK, "uktsp", "tsp", NC_(VOLUME, "British teaspoon"), "1/3", "uktbsp"},
+        {UnitType.DIMENSION, UnitSystem.IMPERIAL_UK, "uksmalltsp", "tsp", NC_(VOLUME, "British small teaspoon"), "1/4", "uktbsp"},
+        {UnitType.DIMENSION, UnitSystem.IMPERIAL_UK, "ukdessertspoon", "dsp", NC_(VOLUME, "British dessertspoon"), "2", "ukteaspoon"},
+        {UnitType.DIMENSION, UnitSystem.IMPERIAL_US, "usgal", "gal|gallon", NC_(VOLUME, "US liquid gallon"), "231", "inch3"},
+        {UnitType.DIMENSION, UnitSystem.IMPERIAL_US, "usqt", "qt|quart", NC_(VOLUME, "US liquid quart"), "1/4", "usgal"},
+        {UnitType.DIMENSION, UnitSystem.IMPERIAL_US, "uspint", "pt|pint", NC_(VOLUME, "US liquid pint"), "1/8", "usgal"},
+        {UnitType.DIMENSION, UnitSystem.IMPERIAL_US, "usfluidounce", "usfloz|floz", NC_(VOLUME, "US fluid ounce"), "1/16", "uspint"},
+        {UnitType.DIMENSION, UnitSystem.IMPERIAL_US, "uscup", "cup", NC_(VOLUME, "US cup"), "8", "usfluidounce"},
+        {UnitType.DIMENSION, UnitSystem.IMPERIAL_US, "uslegalcup", "legalcup|cup", NC_(VOLUME, "US legal cup"), "240/1000", "liter"},
+        {UnitType.DIMENSION, UnitSystem.IMPERIAL_US, "metriccup", "cup", NC_(VOLUME, "US metric cup"), "250/1000", "liter"},
+        {UnitType.DIMENSION, UnitSystem.IMPERIAL_US, "ustablespoon", "ustbsp|tbsp", NC_(VOLUME, "US tablespoon"), "1/16", "uscup"},
+        {UnitType.DIMENSION, UnitSystem.IMPERIAL_US, "uslegaltablespoon", "legaltbsp|tbsp", NC_(VOLUME, "US legal tablespoon"), "1/16", "uslegalcup"},
+        {UnitType.DIMENSION, UnitSystem.IMPERIAL_US, "usteaspoon", "ustsp|tsp", NC_(VOLUME, "US teaspoon"), "1/3", "ustablespoon"},
+
+
+        //Time units
+        {UnitType.TIME, UnitSystem.SI, "second", "sec|s", NC_(TIME, "second"), "1", ""}, // Fundamental
+        {UnitType.TIME, UnitSystem.SI, "minute", "min|m", NC_(TIME, "minute"), "60", "second"},
+        {UnitType.TIME, UnitSystem.SI, "hour", "hr|h", NC_(TIME, "hour"), "60", "minute"},
+        {UnitType.TIME, UnitSystem.SI, "day", "da|d", NC_(TIME, "day"), "24", "hour"},
+        {UnitType.TIME, UnitSystem.SI, "week", "wk", NC_(TIME, "week"), "7", "day"},
+        {UnitType.TIME, UnitSystem.SI, "fortnight", "", NC_(TIME, "fortnight"), "14", "day"},
+        {UnitType.TIME, UnitSystem.SI, "commonyear", "yr|y|calendaryear", NC_(TIME, "Common year"), "365", "day"},
```

Comment:
```suggestion
        {UnitType.TIME, UnitSystem.SI, "commonyear", "calendaryear|year|yr|y", NC_(TIME, "Common year"), "365", "day"},
```
Shouldn't this have  `year` as an abbreviation? 
I would also reorder the abbreviations from long to small like you've done elsewhere.